.github/workflows/build.yml

```diff
@@ -29,23 +29,28 @@ jobs:
         with:
           submodules: recursive
           fetch-depth: 0
+
       - uses: actions/setup-python@v5
         if: ${{ matrix.group == 'storage' }}
         with:
           python-version: "3.9"
+
```

Comment:
Why all these new lines? Is this some standard that we're introducing here? So far, from what I've seen and implemented, our standard is no new-lines in the GA yaml files.

Reply:
Yes, we'd be introducing it here. I find the huge blob without newlines utterly unreadable. We don't code like that either. We can discuss it and I can revert it if you're not happy with it.

Reply:
I personally don't care for as long as we're consistent and follow whatever standard there is about this! No need to revert from my end, but a brief note in #eng-devops-disc would be good as I might just have reverted that in another PR thinking it's unconscious.

Reply:
I personally don't care for as long as we're consistent and follow whatever standard there is about this! No need to revert from my end, but a brief note in #eng-devops-disc would be good as I might just have reverted that in another PR thinking it's unconscious.

Reply:
Let me correct myself, we are already using newlines in several schema modules.

Reply:
I'm not sure, I think with - is in fact fine